README.md

```diff
@@ -59,6 +59,38 @@ Objectives:
 - Extensible: Customizable without touching the core code.
 - Unified: Single codebase, deployable as an agent or collector with support for traces, metrics and logs.
 
+## Community
```

Comment:
Should this be in CONTRIBUTING.md instead?

Reply:
I considered that, but I'd like to keep this open to the community in general, especially the mention of the Slack channel.